tests/apache/livy/triggers/test_livy.py

```diff
@@ -0,0 +1,175 @@
+import asyncio
+from unittest import mock
+
+import pytest
+from aiohttp import ClientConnectionError
+from airflow.providers.apache.livy.hooks.livy import BatchState
+from airflow.triggers.base import TriggerEvent
+
+from astronomer.providers.apache.livy.triggers.livy import LivyTrigger
+
+
+def test_livy_trigger_serialization():
+    """
+    Asserts that the TaskStateTrigger correctly serializes its arguments
+    and classpath.
+    """
+    trigger = LivyTrigger(batch_id=1, spark_params={}, livy_conn_id="livy_default", polling_interval=0)
+    classpath, kwargs = trigger.serialize()
+    assert classpath == "astronomer.providers.apache.livy.triggers.livy.LivyTrigger"
+    assert kwargs == {
+        "batch_id": 1,
+        "spark_params": {},
+        "livy_conn_id": "livy_default",
+        "polling_interval": 0,
+        "extra_options": None,
+        "extra_headers": None,
+        "livy_hook_async": None,
+    }
+
+
+@pytest.mark.asyncio
+@mock.patch("astronomer.providers.apache.livy.triggers.livy.LivyTrigger.poll_for_termination")
+async def test_livy_trigger_run_with_no_poll_interval(mock_poll_for_termination):
+    """
+    Test if the task is run is in triggerr successfully with poll interval=0.
+    :return:
+    """
+    mock_poll_for_termination.return_value = {"status": "success"}
+    trigger = LivyTrigger(batch_id=1, spark_params={}, livy_conn_id="livy_default", polling_interval=0)
+    task = [i async for i in trigger.run()]
+    assert len(task) == 1
+    assert (
+        TriggerEvent({"status": "success", "batch_id": 1, "response": "Batch 1 succeeded", "log_lines": None})
+        in task
+    )
+
+
+@pytest.mark.asyncio
+@mock.patch("astronomer.providers.apache.livy.triggers.livy.LivyTrigger.poll_for_termination")
+async def test_livy_trigger_run_with_poll_interval_success(mock_poll_for_termination):
+    """
+    Test if the task is run is in triggerr successfully with poll interval=30.
+    :return:
```

Comment:
?

Reply:
`Test if the task ran in the triggerer successfully with poll interval>0. In the case when polling_interval > 0, it should return a success or failure status.`

Changing it in the docstring.